shopsafe-frontend/src/app/interfaces/interface.ts

```diff
@@ -9,6 +9,14 @@ export interface ResultInterface {
 export interface StoreResultInterface {
   store: Store[];
   countyStats: CountyStats[];
+  covidData: DataPointInterface[];
+  maskData: DataPointInterface[];
+  
+}
+
+export interface DataPointInterface {
+  value: number;
+  Date: string; // update based on what is returned
```

Comment:
Not sure how youre going to extract the values from the json response but if it makes plotting on charts easier I think you can convert the strings into Date objects by passing in the string that was given. Im changing the dates so that theyre all in the form given in the picture. 
![Screenshot 2020-07-13 at 10 56 50 AM](https://user-images.githubusercontent.com/39624330/87319401-92f94f80-c4f7-11ea-819f-3f7af4f41ef8.png)

Reply:
I think as a String should be okay! But will confirm and let you know